src/bin/edit/draw_menubar.rs

```diff
@@ -107,6 +107,15 @@ fn draw_menu_view(ctx: &mut Context, state: &mut State) {
             tb.set_word_wrap(!word_wrap);
             ctx.needs_rerender();
         }
+
+        if ctx.menubar_menu_checkbox(
```

Comment:
I just noticed that you picked the wrong type here. This should be `menubar_menu_button`. Oh well, I'll fix that later. 🙂